### PR TITLE
Fix coverage

### PR DESCRIPTION
**Pull request by abderrahim**

### CODE REVIEW

File: tests/sourcecache/staging.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def test_staged_source_build(tmpdir, datafiles, cli):
     files = []
     for _, _, filename in os.walk(source_dir):
         files.extend(filename)
-    assert files == []
+    assert not files
```
Comment (nanonyme) [inline tests/sourcecache/staging.py]: Consider `assert not files, files` so we get files into output.

File: tox.ini [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Tox global configuration
 #
 [tox]
-envlist = py37,py{38,39,310}-nocover
+envlist = py37-nocover,py{38,39,310}
```
Comment (juergbi) [inline tox.ini]: Based on the Debian 10 CI run of this branch and https://github.com/apache/buildstream/pull/1420#issuecomment-773094285, coverage should also work with Python 3.7. Have you encountered issues with coverage in Python 3.7?

**Review by juergbi — CHANGES_REQUESTED** [review]
`.github/common.env` needs to be updated to actually use the tox environments with coverage in CI.

Other than that, this looks good to me, assuming CI with coverage enabled will pass.

File: tox.ini (hunk 2 above)
Comment (abderrahim) [reply tox.ini]: I don't remember, I'll try to change `common.env` and see

**Comment by juergbi** [comment]
This was fixed by #1817.

**Pull request closed without merging**